package-res/resources/web/pentaho/visual/base/events/DidUpdate.js

```diff
@@ -18,20 +18,39 @@ define([
 ], function(Event) {
   "use strict";
 
-
+  /**
+   * @name DidUpdate
+   * @memberOf pentaho.visual.base.events
+   * @class
+   * @extends pentaho.lang.Event
+   *
+   * @classDesc This event is emitted when updating the view with
+   * {@link pentaho.visual.base.Model#update|Model#update} occurs without any failures.
+   *
+   * @constructor
+   * @description Creates a `DidUpdate` event.
+   *
+   * @param {!pentaho.visual.base.Model} source - The model object which is emitting the event.
```

Comment:
@param {!pentaho.visual.base.Model} source - The model object that is emitting the event.